plugin/security/src/test/java/org/opencds/cqf/ruler/security/interceptor/AuthenticatorInterceptorIT.java

```diff
@@ -0,0 +1,76 @@
+package org.opencds.cqf.ruler.security.interceptor;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.hl7.fhir.r4.model.Bundle;
+import org.hl7.fhir.r4.model.IdType;
+import org.hl7.fhir.r4.model.MeasureReport;
+import org.hl7.fhir.r4.model.Parameters;
+import org.hl7.fhir.r4.model.StringType;
+import org.junit.jupiter.api.Test;
+import org.opencds.cqf.ruler.cql.CqlConfig;
+import org.opencds.cqf.ruler.cr.CrConfig;
+import org.opencds.cqf.ruler.devtools.DevToolsConfig;
+import org.opencds.cqf.ruler.security.SecurityConfig;
+import org.opencds.cqf.ruler.test.RestIntegrationTest;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import ca.uhn.fhir.rest.server.exceptions.AuthenticationException;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT, classes = {AuthenticatorInterceptorIT.class,
+	SecurityConfig.class, CrConfig.class, CqlConfig.class, DevToolsConfig.class }, properties = {
+	"hapi.fhir.fhir_version=r4", "hapi.fhir.security.security_configuration.enabled=true",
+	"hapi.fhir.security.security_configuration.username=someuser",
+	"hapi.fhir.security.security_configuration.password=thepassword"
+})
+public class AuthenticatorInterceptorIT extends RestIntegrationTest {
+
+	@Test
+	public void testMeasureEvaluateAuth() throws Exception {
```

Comment:
Can we simplify this test to simply be posting/reading data? Adding Measure eval to the test complicates it and could lead to false failures.